tests/test-cases/Group1-Docker-Commands/1-25-Docker-Port-Map.robot

```diff
@@ -12,9 +12,11 @@ Create container with port mappings
     ${rc}  ${output}=  Run And Return Rc And Output  docker ${params} start webserver
     Should Be Equal As Integers  ${rc}  0
     Should Not Contain  ${output}  Error
-    ${rc}  ${output}=  Run And Return Rc And Output  wget ${vch-ip}:10000 --retry-connrefused --timeout=10 --tries=10
```

Comment:
Agreed on the using wait until keyword succeeds, but we also do need to keep this reasonable. Just masking underlying problems with massive timeouts is not the way to go.. if timeout=10 and default 20 tries equals over 3 minutes (I am just guessing, don't know those wget parameters all that well), then something is not right.   I can docker run -d -p 80:80 nginx locally and instantly hit localhost in my browser. It seems like a lot of these "random" CI failures end up being internal component crashes/panics/races that we need to fix and not mask with looser tests.